src/Skoruba.IdentityServer4.Admin/Helpers/Messaging/Interfaces/IEmailSender.cs

```diff
@@ -0,0 +1,26 @@
+#region Assembly Microsoft.AspNetCore.Identity.UI, Version=3.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
```

Comment:
In the STS.Identity project, all of the email message classes Implement the IEmailSenter interface.  This isn't necessary for the Admin project, but instead of pulling in a whole new dependency on Microsoft.AspNetCore.Identity.UI, I thought it more favorable to implement this interface natively.. Otherwise, I would suggest not using this interface / contract at all within the Admin project as it's not necessary.